lms/static/js/edxnotes/views/toggle_notes_factory.js

```diff
@@ -0,0 +1,73 @@
+;(function (define, undefined) {
+'use strict';
+define([
+    'jquery', 'underscore', 'backbone', 'gettext', 'js/edxnotes/views/visibility_decorator'
+], function($, _, Backbone, gettext, EdxnotesVisibilityDecorator) {
+    var ToggleNotesView = Backbone.View.extend({
+        events: {
+            'click .action-toggle-notes': 'toogleHandler'
+        },
+
+        errorMessage: gettext("We couldn't <show/hide> your notes. Make sure that you are connected to the Internet, and then try refreshing the page."),
```

Comment:
@jmclaus Not quite - the text should be "show" or "hide" depending on what the user is trying to do. Is that possible? If not, I would say "We couldn't show or hide your notes. Make sure..."